### ISSUE TITLE
[FEATURE] Display online score manager in Home > Scores

### ISSUE THREAD

**Issue by bkunda** [musescore/MuseScore#16808]
This is a priority task for 4.1.

The idea is to display a logged-in user's available online scores within the MuseScore desktop app. 

A design specification will replace this issue contents soon.

**Comment by jessjwilliamson** [comment]
Designs are ready to be implemented :)
You can find the designs and UX flow here: https://www.figma.com/file/MIOZYp61iPVxutCtbJ2rHc/Home-Tab?type=design&node-id=6-35

<img width="1566" alt="image" src="https://user-images.githubusercontent.com/71656189/236223658-985c48b3-1f93-43fc-b8d4-1b9560f11218.png">

**Comment by shoogle** [comment]
The design features two views: grid view and list view. I suggest the grid view is implemented first and then the list view is done afterwards in a separate task. This way we can get users testing the download aspect sooner.

**Comment by shoogle** [comment]
Blocked by #17444.

![image](https://user-images.githubusercontent.com/11011881/236286825-394d61af-1daa-40e0-886a-c6f806811a9a.png)

**Developers:** Comparison should be done based on the score's revision number rather than its modification date. Use blue if the stored revision number equals the online revision number, otherwise grey.

**Comment by cbjeukendrup** [comment]
See https://github.com/musescore/MuseScore/pull/17764 for a PR that can be considered a very first iteration
